### DISCUSSION TITLE
Cover art - Radio Stations

### DISCUSSION THREAD

**Discussion by clriis** [rern/rAudio#1331] [Bug Reports]
Can no longer change cover art at some radio stations. Nothing happens when I click "Change cover art"

**Comment by ceejayemm** [comment]
Also noted in #1308 

EDIT

This seems to be caused by (as seen using journalctl -xe after trying to select edit coverart):

Jun 01 09:30:45 rAudio-1 nginx[392]: 2023/06/01 09:30:45 [error] 392#392: *379 FastCGI sent in stderr: 
"PHP message: PHP Warning:  Undefined variable $searchmode in /srv/http/mpdlibrary.php on line 469; 
PHP message: PHP Warning:  Undefined variable $searchmode in /srv/http/mpdlibrary.php on line 469;
PHP message: PHP Warning:  Undefined variable $searchmode in /srv/http/mpdlibrary.php on line 469;
PHP message: PHP Warning:  Undefined variable $searchmode in /srv/http/mpdlibrary.php on line 469;
PHP message: PHP Warning:  Undefined variable $searchmode in /srv/http/mpdlibrary.php on line 469; 
PHP message: PHP Warning:  Undefined variable $searchmode in /srv/http/mpdlibrary.php on line 469" 
while reading response header from upstream, client: 192.168.86.53, server: localhost, 
request: "POST /mpdlibrary.php HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm/php-fpm.sock:", 
host: "192.168.86.35", referrer: "http://192.168.86.35/"

Chris
